webapp/src/components/ImageEditor/ImageCanvas.tsx

```diff
@@ -506,7 +505,7 @@ export class ImageCanvasImpl extends React.Component<ImageCanvasProps, {}> imple
     }
 
     protected updateEdit(x: number, y: number) {
-        if (this.edit && this.edit.inBounds(x, y)) {
+        if ((this.edit && this.edit.inBounds(x, y)) || (this.edit && this.isResizing)) {
```

Comment:
```suggestion
        if (this.edit && (this.edit.inBounds(x, y)) || this.isResizing)) {
```

nit; also we might want to consider this for ones like the line / circle / rect tool, it's annoying when you go off canvas slightly and it just... stops

Reply:
yes hahahahhahahaha